crates/swc_core/build.rs

```diff
@@ -12,6 +12,15 @@ compile_error!(
      run plugin, use 'plugin_transform_host_*' instead."
 );
 
+#[cfg(all(feature = "__plugin_transform", feature = "common_concurrent"))]
+compile_error!("plugin transform cannot enable concurrent mode.");
+
+#[cfg(all(feature = "transforms", feature = "transforms_concurrent"))]
```

Comment:
Why are these features are mutually exclusive?

Reply:
Not technically, more for the guide to the users. User do not needs to select transforms / transforms_concurrent both - enabling transforms_concurrent implies to enable transforms already. If they don't need concurrent, using transforms only should work.

Reply:
But what if some of their dependency depend on one feature?

Reply:
> some of their dependency

Do you mean if user level code have another direct deps to the transform? The whole purpose of swc_core is prevent that, user level code should not have another deps which can be a transitive deps to swc_core. If that happens, we'll see more troubles like version conflict before worrying about feature conflict.

Reply:
Hmm... Okay I got it